### ISSUE TITLE
train_trades_mnist.py and train_trades_cifar10.py missing evaluation part

### ISSUE THREAD

**Issue by TalRub104** [Biqing-Qi/Exploring-Adversarial-Robustness-of-Deep-State-Space-Models#3]
Hi,
The scripts train_trades_mnist.py and train_trades_cifar10.py don't include the following:
*Code to save the best checkpoints
*The folllwing evaluation:
`train_loss, train_acc = eval_train(args, model, device, train_loader)
    test_loss, test_acc = eval_test(args, model, device, test_loader)
    adv_test_loss, adv_test_acc = adv_test(args, model, device, test_loader, args.attack_type)`
    
*Call for AA_eval(model, args)

Additionally, I don't understand why the AA_adv_test function doesn't perform adversarial accuracy evaluation as the adv_test function does, nor why it doesn't return any value or print any result.

**Comment by gjq100** [comment]
Hello, you may leave your email address, and we can use it to get in touch. I believe this will facilitate receiving notifications and solving problems more conveniently.

**Comment by TalRub104** [comment]
Hi,
My email is:

talr5@mail.tau.ac.il

Thanks!
